Fix up links identified as being dead or incorrect

shs96c opened the pull request:
In some cases, this means that we've also had to futz with
the spec text, but the logical meaning should be identical.

The W3C link validtor does not correctly check links into
PDF documents, however loading these in Chrome show that
they are valid.

I've removed the test about showModalDialog since this
method has been removed from modern browsers and is no
longer supported.

This does not fix the broken link in the execute script
section

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/w3c/webdriver/791)
<!-- Reviewable:end -->

jgraham commented:
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 3 unresolved discussions.

---

*[webdriver-spec.html, line 294 at r1](https://reviewable.io:443/reviews/w3c/webdriver/791#-KdWSb9EU_zbTyvk2lBk:-KdWSb9EU_zbTyvk2lBl:blggmbk) ([raw file](https://github.com/w3c/webdriver/blob/1cae1a3d57455571730e6e0a221401d3e73fea29/webdriver-spec.html#L294)):*
> ```HTML
>    <!-- Focusing steps --> <li><dfn><a href="https://html.spec.whatwg.org/#focusing-steps">Focusing steps</a></dfn>
>    <!-- Fousable area --><li><dfn><a href=https://html.spec.whatwg.org/#focusable-area>Focusable area</a></dfn>
>    <!-- GetOwnProperty of a Window object --> <li><dfn><a href=https://html.spec.whatwg.org/#windowproxy-getownproperty><code>GetOwnProperty</code> of a <code>Window</code> object</a></dfn>
> ```

`[[GetOwnProperty]]`

---

*[webdriver-spec.html, line 3533 at r1](https://reviewable.io:443/reviews/w3c/webdriver/791#-KdWSEZEIe2zwgjk3W0f:-KdWSEZFAC4c7c44Hgly:b-cfsf0i) ([raw file](https://github.com/w3c/webdriver/blob/1cae1a3d57455571730e6e0a221401d3e73fea29/webdriver-spec.html#L3533)):*
> ```HTML
>      <li><p>Let <var>child window</var> be
>       the <a><code>WindowProxy</code></a> object obtained by
>       calling <a><code>GetOwnProperty</code> of
> ```

I think that's `[[GetOwnProperty]]`

---

*[webdriver-spec.html, line 3535 at r1](https://reviewable.io:443/reviews/w3c/webdriver/791#-KdWSMbP2FkH8kv2Y1IH:-KdWSMbP2FkH8kv2Y1II:b-ir5o18) ([raw file](https://github.com/w3c/webdriver/blob/1cae1a3d57455571730e6e0a221401d3e73fea29/webdriver-spec.html#L3535)):*
> ```HTML
>       calling <a><code>GetOwnProperty</code> of
>       a <code>Window</code> object</a> <var>window</var> with
>       index <var>id</var>.
> ```

s/index/argument/

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/w3c/webdriver/791)*
<!-- Sent from Reviewable.io -->

shs96c commented:
Review status: all files reviewed at latest revision, 3 unresolved discussions.

---

*[webdriver-spec.html, line 294 at r1](https://reviewable.io:443/reviews/w3c/webdriver/791#-KdWSb9EU_zbTyvk2lBk:-KdWUNQ215PUpzINe2jd:b-886gsx) ([raw file](https://github.com/w3c/webdriver/blob/1cae1a3d57455571730e6e0a221401d3e73fea29/webdriver-spec.html#L294)):*
<details><summary><i>Previously, jgraham wrote…</i></summary><blockquote>

`[[GetOwnProperty]]`
</blockquote></details>

Fixed

---

*[webdriver-spec.html, line 3533 at r1](https://reviewable.io:443/reviews/w3c/webdriver/791#-KdWSEZEIe2zwgjk3W0f:-KdWUOl6WjJ2XrRwJPnk:b5u6if4) ([raw file](https://github.com/w3c/webdriver/blob/1cae1a3d57455571730e6e0a221401d3e73fea29/webdriver-spec.html#L3533)):*
<details><summary><i>Previously, jgraham wrote…</i></summary><blockquote>

I think that's `[[GetOwnProperty]]`
</blockquote></details>

Also fixed

---

*[webdriver-spec.html, line 3535 at r1](https://reviewable.io:443/reviews/w3c/webdriver/791#-KdWSMbP2FkH8kv2Y1IH:-KdWUPffaRgxAT8o5rvg:b-886gsx) ([raw file](https://github.com/w3c/webdriver/blob/1cae1a3d57455571730e6e0a221401d3e73fea29/webdriver-spec.html#L3535)):*
<details><summary><i>Previously, jgraham wrote…</i></summary><blockquote>

s/index/argument/
</blockquote></details>

Fixed

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/w3c/webdriver/791)*
<!-- Sent from Reviewable.io -->

jgraham commented:
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/w3c/webdriver/791)*
<!-- Sent from Reviewable.io -->
